src/routes/NewUser/components/NewUserPage/NewUserPage.js

```diff
@@ -1,24 +1,30 @@
 import React, { useState } from 'react';
-import Typography from '@material-ui/core/Typography';
-import TextField from '@material-ui/core/TextField';
-import Paper from '@material-ui/core/Paper';
-import Divider from '@material-ui/core/Divider';
-import Button from '@material-ui/core/Button';
-import Card from '@material-ui/core/Card';
-import CardActions from '@material-ui/core/CardActions';
-import CardContent from '@material-ui/core/CardContent';
-import Container from '@material-ui/core/Container';
-import Grid from '@material-ui/core/Grid';
+import {
```

Comment:
I have found that the single imports are good to do for smaller bundle sizes since some packages require a webpack plugin to correctly be tree shaken - not sure that is the case here, but just my tendency

Reply:
Yes, this has been on my list to look into.  My quick search a week ago made me conclude that create-react-app already had the necessary pieces, but I haven't taken the time to verify it.